lib/models/updateAction.js

```diff
@@ -188,8 +188,9 @@ function doRequest(action, event, token, callback) {
             }
             if (!err && data.body && data.body.orionError) {
                 err = new errors.OrionError(JSON.stringify(data.body.orionError));
+            } else {
+                callback(err, data);
             }
-            callback(err, data);
```

Comment:
I think this change is no correct... the callback() needs to be changed in any case and, after this change, it is only called when the execution flow goes by the "else" part.

Reply:
Fixed in c29e89f6dc2c5b39008fa8a025d1cf01573c17e3